src/Worker/Targets/netstandard1.0/Microsoft.NET.Sdk.Worker.targets

```diff
@@ -0,0 +1,26 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Worker.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <EnableDefaultContentItems Condition=" '$(EnableDefaultContentItems)' == '' ">true</EnableDefaultContentItems>
+    <RunWorkingDirectory Condition=" '$(RunWorkingDirectory)' == '' and '$(EnableDefaultRunWorkingDirectory)' != 'false' ">$(MSBuildProjectDirectory)</RunWorkingDirectory>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectCapability Include="DotNetCoreWorker" />
+    <ProjectCapability Include="SupportHierarchyContextSvc" />
+    <ProjectCapability Include="DynamicDependentFile" />
+    <ProjectCapability Include="DynamicFileNesting" />
```

Comment:
>DynamicFileNesting [](start = 32, length = 18)

These two capabilities enable nesting feature but not bringing you default nesting settings, thus users would be able to add new nesting settings bit would not have defaults. If you need defaults like web had add this capability: "WebNestingDefaults"

Reply:
We're trying to avoid any web dependencies. There's already an issue for getting the nesting behavior decoupled from the web components. https://devdiv.visualstudio.com/DevDiv/_workitems/edit/801266

Reply:
This is exactly what i am talking about. The issue you mentioned added this capability so others sdks could choose to have defaults similar to wed defaults. Issue is already resolved.

---
In reply to: [261335786](https://github.com/aspnet/websdk/pull/498#discussion_r261335786) [](ancestors = 261335786)

Reply:
Fair. That new name is misleading.